src/lib.rs

```diff
@@ -9,6 +9,7 @@
 //! the STM32L432KC microcontroller. Participation is of course very welcome!
 
 #![no_std]
+#![allow(clippy::upper_case_acronyms)]
```

Comment:
This was causing peripheral / register defines to raise clippy warnings. It has only in a single module, so could move this to only the scope of that module potentially?